src/app/config.service.ts

```diff
@@ -28,19 +28,40 @@ export class ConfigService {
    *
    * @return {boolean}
    */
-  useHttpdServer(): boolean {
-    const value = get(this.config, 'useHttpdServer');
+  useHttpServer(): boolean {
+    const value = get(this.config, 'contentSearch.enable');
     return isBoolean(value) ? value : value === 'true';
   }
 
+  /**
+   * Return server url value, set from json file config.json
+   *
+   * @returns {string}
+   */
+  getUrlServer(): string {
+    const value = get(this.config, 'contentSearch.url');
+    return value.length > 0 ? value : '';
+  }
+
   /**
    * Return exact-match value, set from json file config.json
    * Both string and boolean value types are allowed
    *
    * @return {boolean}
    */
   useExactMatch(): boolean {
-    const value = get(this.config, 'search.exactMatch');
+    const value = get(this.config, 'contentSearch.exactMatch');
+    return isBoolean(value) ? value : value === 'true';
+  }
+
+  /**
+   * Return match case value, set from json file config.json
+   * Both string and boolean value types are allowed
+   *
+   * @return {boolean}
+   */
+  useMatchCase(): boolean {
+    const value = get(this.config, 'contentSearch.matchCase');
     return isBoolean(value) ? value : value === 'true';
```

Comment:
You can compare with the operator ==
https://developer.mozilla.org/fr/docs/Web/JavaScript/Equality_comparisons_and_sameness#l%C3%A9galit%C3%A9_faible_avec